src/templates/narratives/index.html

```diff
@@ -1,11 +1,15 @@
-{% extends template %}
+{% extends template %} 
```

Comment:
There is some more trailing whitespace in this file.

Reply:
Now end-of-line spaces should be gone from this file.